### ISSUE TITLE
updating error

### ISSUE THREAD

**Issue by wangtengda0310** [ipfs-shipyard/ipfs-desktop#1546]
Please describe what you were doing when this error happened.

**Specifications**

- **OS**: win32
- **IPFS Desktop Version**: 0.11.2
- **Electron Version**: 8.2.4
- **Chrome Version**: 80.0.3987.165

**Error**

```
TypeError: stopIpfs is not a function
    at quitAndInstall (D:\Program Files\IPFS Desktop\resources\app.asar\src\auto-updater\quit-and-install.js:13:24)
    at Immediate.<anonymous> (D:\Program Files\IPFS Desktop\resources\app.asar\src\auto-updater\index.js:87:9)
    at processImmediate (internal/timers.js:439:21)
```

**Comment by wangtengda0310** [comment]
![image](https://user-images.githubusercontent.com/17254793/84219543-61115980-ab03-11ea-9d9a-fa8e5293489c.png)
![image](https://user-images.githubusercontent.com/17254793/84219554-67073a80-ab03-11ea-9da7-e5bf14676989.png)

**Comment by rafaelramalho19** [comment]
This was fixed with #1509, please make sure you update to the latest version now 😄
